API/API.mli

```diff
@@ -584,13 +569,11 @@ sig
     val fold_outside : (('c, 't) Declaration.pt -> 'a -> 'a) -> ('c, 't) pt -> init:'a -> 'a
 
     (** Return the set of all identifiers bound in a given named-context. *)
-    val to_vars : ('c, 't) pt -> Names.Id.Set.t
 
     (** [to_instance Ω] builds an instance [args] such
       that [Ω ⊢ args:Ω] where [Ω] is a named-context and with the local
       definitions of [Ω] skipped. Example: for [id1:T,id2:=c,id3:U], it
       gives [Var id1, Var id3]. All [idj] are supposed distinct. *)
-    val to_instance : (Names.Id.t -> 'r) -> ('c, 't) pt -> 'r list
```

Comment:
OK, but I think it makes sense to remove also the accompanying comment from `API/API.mli` file.